packages/components/tests/dummy/app/templates/components/power-select/after-options.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="hds-power-select_after-options">
```

Comment:
Is this file needed? 🤔

Reply:
We use this to exemplify how a custom component/template block can be passed in. It's called in [one of the examples](https://github.com/hashicorp/design-system/pull/539/files/4c181a89655938a0d004db044bb82273140f0f12#diff-ac16ea0a6f51865dfe0a911f35693d355234436b89274cad9807836ab60e3a89R98).

Reply:
OK. I can't think of better alternatives, so let's leave it like this.